app/controllers/admin/airlines_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Admin::AirlinesController < ApplicationController
```

Comment:
This is a solid well factored controller. If these controller methods get any bigger or more complex you'll want to start wrapping them in Controller unit tests (feel free to reach out for how to write those) and/or extracting their logic into the models or Plain old ruby objects.